src/appellate/utils.js

```diff
@@ -48,12 +92,53 @@ let APPELLATE = {
 
   // Create a list of doc_ids from the list of all links available on the page
   findDocLinksFromAnchors: (nodeList) => {
-    const links = [];
+    let links = [];
+    let docsToCases = {};
     Array.from(nodeList).map((a) => {
-      if (a.title !== 'Open Document') return;
+      if (!PACER.isDocumentUrl(a.href)) return;
+
+      // this regex will match the doc_id and case_id passed as an argument in the
+      // onclick event of each anchor element related to a document
+      let doDocPost = /^return doDocPostURL\('([^']*)','([^']*)'\);/.exec(a.getAttribute('onclick'));
+      let params = {};
+      if (doDocPost) {
+        [, params.doc_id, params.case_id] = doDocPost;
+        docsToCases[params.doc_id] = params.case_id;
+      }
```

Comment:
This can easily go in a helper function, where it'll be a lot easier to test.

Reply:
You're right! I will create the helper function and add new tests,